src/components/styledWrapper.jsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+const styledWrapper = styled.div`
+	width: 100vw;
+	height: 100vh;
+`;
+
+export default styledWrapper;
```

Comment:
Same as the file name, the name of your styled component should be written in PascalCase.

```suggestion
const StyledWrapper = styled.div`
	width: 100vw;
	height: 100vh;
`;

export default StyledWrapper;
```